web/src/components/users/FirstUser.jsx

```diff
@@ -220,6 +253,29 @@ export default function FirstUser() {
                 isRequired
                 onChange={handleInputChange}
               />
+              { showSuggestions && formValues.fullName && !formValues.userName &&
+                <Menu
+                  aria-label={_("Username suggestion dropdown")}
+                  className="first-username-dropdown"
+                  onMouseEnter={() => setInsideDropDown(true)}
+                  onMouseLeave={() => setInsideDropDown(false)}
+                >
+                  <MenuContent>
+                    <MenuList>
+                      { suggestUsernames(formValues.fullName).map((suggestion, index) => (
+                        <MenuItem
+                          key={index}
+                          itemId={index}
+                          onClick={() => { setInsideDropDown(false); setFormValues({ ...formValues, userName: suggestion }) }}
+                        >
+                          { /* TRANSLATORS: dropdown username suggestions */ }
+                          { _("Use suggested username ") }
+                          <strong>{suggestion}</strong>
```

Comment:
I believe it's better to go for a `<b>` HTML element here instead. 

* https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong
* https://html.spec.whatwg.org/#the-b-element
  > The **b** element represents a span of text to which attention is being drawn for utilitarian purposes **without conveying any extra importance** and with no implication of an alternate voice or mood